packages/node_modules/nav-frontend-lastestriper/src/lastestriper.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import * as PT from 'prop-types';
+import 'nav-frontend-lastestriper-style';
+
+export interface NavFrontendLastestriperProps {
+    /**
+     * Antall lastestriper. Default er 3.
+     */
+    linjer?: number;
+    /**
+     * Inline stiling
+     */
+    style?: object;
```

Comment:
Tror jeg ville fjernet `style` og `className` tilfordel for å arve fra `HTMLDivElement` sine props.

Reply:
Done. Vet ikke helt hva man gjør med proptypes seksjonen. Det står fortsatt classname og style definisjoner der.